schema.json

```diff
@@ -1,28 +1,55 @@
 {
   "type": "object",
-  "required": ["apiName","targetHost", "targetPort","allowablePeakMemoryVariance",
-    "numIterations","allowableServiceResponseTimeVariance","testCaseDir",
-    "testSuiteDir","baseStatsOutputDir","reportOutputDir","concurrentUsers",
-    "testSuite","requestDelay","TPSFreq","rampUsers","rampDelay"],
-  "properties":
-  {
-    "apiName": {"type": "string"},
-    "targetHost":{"type": "string"},
-    "targetPort":{"type": "string"},
-    "numIterations":{"type": "integer","minimum": 0 },
-    "allowablePeakMemoryVariance":{"type": "number","minimum": 0},
-    "allowableServiceResponseTimeVariance":{"type": "number","minimum": 0},
-    "testCaseDir":{"type": "string"},
-    "testSuiteDir":{"type": "string"},
-    "baseStatsOutputDir":{"type": "string"},
-    "reportOutputDir":{"type": "string"},
-    "concurrentUsers":{"type": "integer","minimum": 0},
-    "testSuite":{"type": "string"},
-    "memoryEndpoint":{"type": "string"},
-    "requestDelay":{"type": "integer","minimum": 0},
-    "TPSFreq":{"type": "integer","minimum": 0},
-    "rampUsers":{"type": "integer","minimum": 0},
-    "rampDelay":{"type": "integer","minimum": 0}
+  "required": [
+    "apiName",
+    "targetHost",
+    "targetPort",
+    "numIterations",
+    "concurrentUsers",
+    "allowablePeakMemoryVariance",
+    "allowableServiceResponseTimeVariance",
+    "testSuite",
+    "requestDelay",
+    "TPSFreq",
+    "rampUsers",
+    "rampDelay",
+    "testCaseDir",
+    "testSuiteDir",
+    "baseStatsOutputDir",
+    "reportOutputDir"
+  ],
+  "properties": {
+    "apiName": { "type": "string" },
+    "targetHost": { "type": "string" },
+    "targetPort": {
+      "type": "string",
+      "minimum": 1,
+      "maximum": 65535
+    },
+    "memoryEndpoint": { "type": "string" },
+    "numIterations": { "type": "integer", "minimum": 0 },
+    "concurrentUsers": { "type": "integer", "minimum": 0 },
```

Comment:
use the requirements to build these minimum values. Anywhere it says "positive integer", use a minimum of 1. I don't know how nicely defaults work, but there are some defaults in the story and ye can come up with sensible defaults too.